tests/unit/ovirtlago/test_runtest.py

```diff
@@ -19,12 +19,15 @@ def dummy_test():
     return DummyTest(id=lambda: 123)
 
 
-@pytest.mark.parametrize('exc', [ExtractPathError, ExtractPathNoPathError])
+@pytest.mark.parametrize(
+    ('exc', 'remote_path'),
```

Comment:
Are you sure that you can put tuple in here?
https://docs.pytest.org/en/latest/parametrize.html

Reply:
argnames – a comma-separated string denoting one or more argument names, or a list/tuple of argument strings.